### PR TITLE
Dev5

### PR DESCRIPTION
**Pull request by khachaneojas**
LOCAL PORT changed

### CODE REVIEW

File: JS/script.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const HttpMethod = {
 };
 
 const BASE_URL = {
-    LOCAL: `http://localhost:8888`,
+    LOCAL: `http://localhost:1001`,
```
Comment (st0rm1O1) [inline JS/script.js:9]: Why? changing the Port from 8888 to 1001??

**Review by st0rm1O1 — COMMENTED** [review]
Need to be checked for Port of Local.

File: JS/script.js (hunk 1 above)
Comment (khachaneojas) [reply JS/script.js:9]: Changes required

Comment (st0rm1O1) [reply JS/script.js:9]: OK!

**Comment by khachaneojas** [comment]
Noted

**Pull request merged**